src/rustdoc-json-types/lib.rs

```diff
@@ -30,7 +30,7 @@ pub type FxHashMap<K, V> = HashMap<K, V>; // re-export for use in src/librustdoc
 /// This integer is incremented with every breaking change to the API,
 /// and is returned along with the JSON blob as [`Crate::format_version`].
 /// Consuming code should assert that this value matches the format version(s) that it supports.
-pub const FORMAT_VERSION: u32 = 46;
+pub const FORMAT_VERSION: u32 = 47;
```

Comment:
This should be rebased on top of #142601. There are several PR's that all want to make the 46->47 change, but I don't need to keep track of all them myself now, git merge-conficts will do that for me!!